fastlane/Fastfile

```diff
@@ -119,6 +122,8 @@ platform :android do
   end
 
   lane :finalize_hotfix_release do |options|
+    push_to_git_remote()
```

Comment:
```suggestion
```

This is not necessary since we are not changing anything in the project before this point.